continuum/environment.yml

```diff
@@ -3,12 +3,12 @@ channels:
 dependencies:
 - lammps =2021.07.30=*mpich*
 - damask =3.0.0a5
-- fenics =2019.1.0
+- fenics =2019.1.0=*_26
```

Comment:
We have to check that the last docker image with worked successful used this build version. This can be checked in the image using `conda list fenics`.

Reply:
```
fenics                    2019.1.0        py39hf3d152e_24    conda-forge
```

Reply:
Other libraries which were tricky in the past:
```
superlu                   5.2.2                h16cfea0_0    conda-forge
superlu_dist              6.2.0                h25dcc4a_5    conda-forge
```